.gitlab/deception.gitlab-ci.yml

```
@@ -0,0 +1,25 @@
+
+cleanup:
+ stage: .pre
+ extends:
+  - .SVC-Account-Cleanup
```

```suggestion
 extends: .SVC-Account-Cleanup
```
Just an FYI that this syntax works to save a single line here and there - purely cosmetic